clients/src/test/java/org/apache/kafka/common/requests/PushTelemetryRequestTest.java

```diff
@@ -34,4 +53,65 @@ public void testGetErrorResponse() {
         assertEquals(Collections.singletonMap(Errors.CLUSTER_AUTHORIZATION_FAILED, 1), response.errorCounts());
     }
 
+    @ParameterizedTest
+    @EnumSource(CompressionType.class)
+    public void testMetricsDataCompressionTypeNone(CompressionType compressionType) {
```

Comment:
It's slightly odd that it's a parameterized test, although it's `CompressionTypeNone`.

Reply:
My bad, moved multiple methods to single method with parametrized test and missed the method name correction. Done.